### ISSUE TITLE
[Tropical Biome]Plants to add

### ISSUE THREAD

**Issue by kaptain-kavern** [kaptain-kavern/EcosystemOverhaul#6]
| Plants |  |  |  |
| :-: | :-: | :-: | :-: |
| Fern 1,2,3 | tall grass stats | Slow movement | _Must be edible by herbivores_ |
| Palm01 | tall grass stats | slow movement (like tall grass) | _Must be edible by herbivores_ |
| Palm02 | minyield: 4/ max: 22/ growth time: 12 | HP: 120/ beauty: 1 | __ |
| Palm03 | minyield: 6/ max: 32/ growth time: 18 | HP: 210/ beauty: 1 | __ |
| Strangler Fig | minyield: 10/ max: 18/ growth time: 10 | HP: 300/ beauty: 2 |  |
| Mahogany Tree | minyield: 22/ max: 56/ growth time: 34 | HP: 320/ beauty: 2 |  |
| Rafflesia | bush stats | -15 beauty | __ |

---
- [x] ~~Fern (3 variations)~~
- [x] ~~Palm (3 variations)~~
- [x] ~~Mahogany Tree~~
- [x] ~~Strangler Fig~~
- [x] Rafflesia

---

[Spreadsheet with all plants Data extracted from game files (for balancing values)](https://docs.google.com/spreadsheets/d/17mnvNxKKrr1btSPEnfVH6m_6lMO6sthmJkeL1HL3BaI/edit?usp=sharing)

**Comment by lpCoercion** [comment]
For the plants, I will send a spreadsheet, but I was thinking the following:

| Plants |  |  |  |
| --- | --- | --- | --- |
| Fern 1,2,3 | tall grass stats | Slow movement | _Must be edible by herbivores_ |
| Bromeliad | tall grass stats | slow movement (like tall grass) | _Must be edible by herbivores_ |
| Palm01 | tall grass stats | slow movement (like tall grass) | _Must be edible by herbivores_ |
| Palm02 | minyield: 4/ max: 22/ growth time: 12 | HP: 120/ beauty: 1 | __ |
| Palm03 | minyield: 6/ max: 32/ growth time: 18 | HP: 210/ beauty: 1 | __ |
| Strangler Fig | minyield: 10/ max: 18/ growth time: 10 | HP: 300/ beauty: 2 |  |
| Mahogany Tree | minyield: 22/ max: 56/ growth time: 34 | HP: 320/ beauty: 2 |  |
| Bamboo | bush stats | beauty: 0 | _I still struggle to make the Panda eating specifically it_ |
| Taro | minyield: 2/ max: 6/ growth time: 2 | HP: 100/ beauty: 0 | _Must drop edible Taro root_ |
| Rafflesia | bush stats | -15 beauty | _I wish we could find something positive to balance the beauty debuff  : ?maybe drug/medicine ingredients?_ |
| Coffee | minyield: 6/ max: 10 | growth time: 3/ HP: 120/ beauty: -1 | _Must drop drug-like coffee ingredient_ |

---

_Edit: Just updated your comment for doing a table with it and add my thoughts in _italic__

**Comment by kaptain-kavern** [comment]
What the min/max value means in this context please?

Edit : Oh maybe their lifespan?

**Comment by lpCoercion** [comment]
min/max is resource drop (e.g. wood, food, etc.)

I do not think that these palms should drop coconuts so that it is balanced... but we could add a rarer palm that drops coconuts.

I will add lifespans.

I will think about a positive for the rafflesia.

**Comment by kaptain-kavern** [comment]
Ok doke

**Comment by lpCoercion** [comment]
I think we should leave the rafflesia with just the debuff. It adds something different; adds another challenge; its a rare spawn; doesnt reward the player and is instead makes the player prioritize the debuff

**Comment by kaptain-kavern** [comment]
I was thinking of a way to "force" the player to deal with the debuff bonus :p but hey no problem we have more important things to focus to :stuck_out_tongue_closed_eyes:

**Comment by kaptain-kavern** [comment]
Have you planned on doing Leafless version of your new plants? Because on Vanilla there is, for plants and trees.

|  |  |  |
| :-: | :-: | :-: |
| Plant | "normal" | leafless |
| Oak | ![treeoaka](https://cloud.githubusercontent.com/assets/2794612/18411985/74ced5b2-7784-11e6-8944-93cf3980d989.png) | ![treeoak_leaflessb](https://cloud.githubusercontent.com/assets/2794612/18411984/74cc8e74-7784-11e6-8ccf-ce06bae61de9.png) |
| Oak | ![treeoakb](https://cloud.githubusercontent.com/assets/2794612/18411983/74cc4ec8-7784-11e6-9673-fd88ce8ca896.png) | ![treeoak_leaflessa](https://cloud.githubusercontent.com/assets/2794612/18411986/74d0667a-7784-11e6-9bd1-cf8928149c67.png) |
| Bush | ![bush](https://cloud.githubusercontent.com/assets/2794612/18412010/28037f84-7785-11e6-82e3-86a254a52ede.png) | ![bush_leafless](https://cloud.githubusercontent.com/assets/2794612/18412009/2800901c-7785-11e6-951f-470388a09a2c.png) |

Not urgent IMHO, I think I only see them in game  when plants die from cold

**Comment by kaptain-kavern** [comment]
Also added link to a [Spreadsheet with all plants Data extracted from game files (for balancing values)](https://docs.google.com/spreadsheets/d/17mnvNxKKrr1btSPEnfVH6m_6lMO6sthmJkeL1HL3BaI/edit?usp=sharing)

**Comment by lpCoercion** [comment]
Yeah I agree. Trees wont lose leaves in the rainforest.

and cool. I will work on plant values

**Comment by lpCoercion** [comment]
I should be able to work on a lot of textures tomorrow :)

I have a few more plants that I want to add for variety and color:

Coconut Palm: Coconuts (edible)
Banana Tree: Bananas (edible)
Heliconia: (tall grass stats/ 3 beauty)

**Comment by kaptain-kavern** [comment]
Oh cool, now that the mod is called "Overhaul" it's good we add more :p 

I'm not sure I can make the Coconut drop two different resources, I will have to test :wink:

**Comment by lpCoercion** [comment]
Strangler Fig

![treestranglerfig](https://cloud.githubusercontent.com/assets/22029252/18436646/792f76a4-78c7-11e6-9d9d-c1442d8be72d.png)

Mahogany

![treemahogany](https://cloud.githubusercontent.com/assets/22029252/18436653/80f32a66-78c7-11e6-9292-8c5456675ad0.png)

Taro
![taroplant](https://cloud.githubusercontent.com/assets/22029252/18437141/887db66e-78c9-11e6-8bec-02bab8f5630b.png)![tarofood](https://cloud.githubusercontent.com/assets/22029252/18437142/8bb4fbf8-78c9-11e6-8fb4-3427d2a9198c.png)

**Comment by kaptain-kavern** [comment]
All in (on my PC, will upload soon)

**Comment by lpCoercion** [comment]
Banana

![treebanana](https://cloud.githubusercontent.com/assets/22029252/18444790/af9c4e58-78e8-11e6-9ed4-94ed5e850994.png)

Rafflesia

![rafflesia](https://cloud.githubusercontent.com/assets/22029252/18449486/5c8840e0-78fd-11e6-9fdf-f98feb336d6c.png)

Bromeliad

![bromeliad](https://cloud.githubusercontent.com/assets/22029252/18450040/8898944e-78ff-11e6-836f-efb15fd5d2e8.png)

**Comment by lpCoercion** [comment]
I removed lemongrass... 

I think we should remove coffea too... it may not look right with the other plants.. what are your thoughts?

**Comment by kaptain-kavern** [comment]
I'm actually adding those three.

I've notice that the files you post have not all the same size. Is that intended ? I mean did you do it on purpose regarding to sizes intended for plants or trees? I don't know if that mean anything but Vanilla textures are all 64x64, 128x128 or 256x256 apparently

**Comment by kaptain-kavern** [comment]
About lemongrass and coffee I don't see reasons for removing them, but I'm all for adding a lot of plants and animals, but hey i'm not the one that need to draw ;) 
Maybe you have things in mind I don't see/know and need to remove them

**Comment by lpCoercion** [comment]
Well I can come back to them last and see how we can tie them into everything.

and kind of. I have been using 64x64 for small stuff and 128x128 for larger things

**Comment by kaptain-kavern** [comment]
Ok because I had some with something like 77*77 (for Taro Roots and plants IIRC).

On another subject : https://github.com/kaptain-kavern/EcosystemOverhaul/commit/6a5273b4c6a354f1b68faa8169624106b307b139 - I just uploaded the last version with all the new plants. I use a free .png find on the web for the Banana Fruit. 

I'm not satisfied yet on how plants ratio are balanced when the game is running (too much Rafflesia, not enough Bromeliad etc) but it just a matter of heavily testing while changing values ;) 

Here is a screenshot. Those damn Rafflesias, i can't managed to make them pop alone (they always are in groups, i'm still searching ^^ )
![screenshot](http://i.imgur.com/srA4JRe.jpg)

**Comment by lpCoercion** [comment]
It is looking great :)

do you need me to fix the size of those plants?

I will get you the: Bromeliad02, Heliconia, Bamboo, Coconut Palm, Coconut Edible and Banana Edible next

and then start working on the animals

Once we have all of the plants in and the weather figured out, I will work on matching/adjusting the plant colors (e.g. I may make the Mahogany and Palm trunks darker to match the Strangler Fig).

**Comment by lpCoercion** [comment]
btw.. palm02 is too small

**Comment by kaptain-kavern** [comment]
All right mate. Fine by me. 

I'm pretty sure that my screenshots have messed-up colors one time out of two (I'm using a software to auto post on Imgur and I think it is the culprit) so please be sure to try this on your PC before taking decision about the colors ;).

Ok doke for the palm02.  
I'll try to change from 

```
<visualSizeRange>
        <min>0.5</min>
        <max>0.9</max>
</visualSizeRange>
```

to

```
<visualSizeRange>
        <min>0.8</min>
        <max>1.2</max>
</visualSizeRange>
```

**Comment by lpCoercion** [comment]
alright cool :) I dont mind the bright colors.

I will take a look at the palm size.

any luck with the weather?

**Comment by lpCoercion** [comment]
also, is it possible to add the vanilla rice plant as a wild plant in the rainforest?

**Comment by kaptain-kavern** [comment]
yeah I was thinking the same about smokeleaf (marijuana) and I envisaged rice too and cotton. We are kinda synchronized hehe :p 

So yes it's possible I started to do it yesterday. We "just" have to redefine the plant def in our mods and adding the correct <sowTag>.

**Comment by lpCoercion** [comment]
I thought about smokeleaf too, but I think it grows wild more in a lowland middle eastern climate instead of the rainforest.

I dont think we should add cotton, because it may be hard to balanace gameplay and also may be slightly off climate wise.

I did also think about healroot, but that may also be unbalanced... and is a made up plant.

**Comment by kaptain-kavern** [comment]
![Imgur](http://i.imgur.com/vVOVF0z.jpg)

Screenshot from my previous try

**Comment by lpCoercion** [comment]
those look pretty good :)

maybe we should include wild smokeleaf, cotton, rice and healroot

I think the palm02 still needs to be bigger though.. maybe half the size of a palm tree?

**Comment by lpCoercion** [comment]
thos look really good :)

What are your thoughts on adding psycoid plant? would it fit in the rainforest? I dont know the description of it.

**Comment by kaptain-kavern** [comment]
To me Psycoid plants in game are inspired by [Coca plant](https://en.wikipedia.org/wiki/Coca). And Wikipedia says " _well adapted to the eastern Andes of Peru and Bolivia, an area of humid, tropical, montane forest._" 
So to me it's OK ^^

**Comment by lpCoercion** [comment]
alright cool lets add that too.. how are the uploads coming? just waiting on textures?

**Comment by kaptain-kavern** [comment]
Nearly yes. I just had the pleasure to be granted with a tamed Gorilla for start in my last play ^^ Too bad they're ugly for now :monkey_face: 

I'm currently testing (by trying to play for real with our mod and some others) and adjusting values as I play. I find the repartition of plants in the wild more balanced now (maybe still too much taro - see picture lower), animals too . 
![Imgur](http://i.imgur.com/Q1yLn8U.jpg)
I now focus my attention a bit on plants value in term of nutrition, growing speed and all of that :stuck_out_tongue:

**Comment by lpCoercion** [comment]
Looks awesome :)

We probably want a fast growing speed due to the nature of the rainforest.

**Comment by lpCoercion** [comment]
I should have the rest of the plants and edibles to you tonight: Heliconia, Coconut Palm, Coffea, Coconuts, Bananas, Coffee Beans

After that I may rebalance some of the plant art. mainly:

All Trees: Add ferns around bases
Mahogany Tree (darker trunks, bolder trunk outline, brighter leaves, more detailed roots)
Palm Tree (more brown in trunks)
Bromeliad: different green (more saturated green)

**Comment by kaptain-kavern** [comment]
Awesome!
I start with their codes now then ;)

---

If I may suggest to also do something about the taro roots (not urgent matter too). 
![taroroot](https://cloud.githubusercontent.com/assets/2794612/18590391/f2187b8c-7c2e-11e6-97da-899c2501f0cb.png)
They are so small that they are hard to distinguish on floors (even worse on forest floors) couldn't you add a black outline or draw them bigger or draw several in a box/bag/crate (like with vanilla harvested raw food?) or something else (i'm clueless when it's about drawing sorry) 
![rice](https://cloud.githubusercontent.com/assets/2794612/18590460/2d3141cc-7c2f-11e6-80fc-b9007b9cbbb6.png) ![potatoes](https://cloud.githubusercontent.com/assets/2794612/18590337/b46804e2-7c2e-11e6-97e8-dbaf7ba32411.png) ![carrot](https://cloud.githubusercontent.com/assets/2794612/18590564/a239b738-7c2f-11e6-93b9-be5c600d1ec4.png)![banana](https://cloud.githubusercontent.com/assets/2794612/18590801/7f169464-7c30-11e6-80ad-53b653b1aa90.png)![rawcoffee](https://cloud.githubusercontent.com/assets/2794612/18590593/c65fcde6-7c2f-11e6-9be0-59adb5157a2c.png) 
_Some inspirations_

**Comment by lpCoercion** [comment]
Yea. I hadnt seen them in game so I wasnt sure how they would look but ill figure something out :)

**Comment by lpCoercion** [comment]
is there a way to make it so the plants dont sway? or at least move a lot less?

**Comment by lpCoercion** [comment]
also, are you able to try and make the plants a little bit bigger... so like trees are huge and ferns kind of overlap a bit? This may help reduce the dirt a bit :)

**Comment by lpCoercion** [comment]
I made some changes to the plants to better fill out the dead space of the jungle and match the color palette:

Palm Tree 02
![palm02](https://cloud.githubusercontent.com/assets/22029252/18610160/ce172544-7ce2-11e6-860b-654f13ecc7f2.png)

Palm Tree 03
![palm03](https://cloud.githubusercontent.com/assets/22029252/18610150/7b72229e-7ce2-11e6-8bd2-7542a6ce9146.png)

Mahogany Tree
![treemahogany](https://cloud.githubusercontent.com/assets/22029252/18610137/391763f0-7ce2-11e6-972b-3ae2a8d396fd.png)

Strangler Fig
![treestranglerfig](https://cloud.githubusercontent.com/assets/22029252/18610139/39179686-7ce2-11e6-8b10-9e6a5ac74ba8.png)

**Comment by lpCoercion** [comment]
also, a new Taro Plant to replace:

![taroplant](https://cloud.githubusercontent.com/assets/22029252/18610201/d51ca9ee-7ce3-11e6-9027-f86db2f3efaf.png)

**Comment by kaptain-kavern** [comment]
OMG even more awesome! gratz buddy
So i will implement them right now.

**About the plants swaying :**  I really don't know if it's possible (XML wise). I will look into that.

**About plant sizes :** you want me to make all of them bigger or only the trees ?

**Comment by lpCoercion** [comment]
plants swaying: i dont think it really matters now that I think about it... the effect makes me nausous but there is an option in game to turn it off lol

I think all of the plants should be bigger. The trees should  feel really big...if we can pull it off... and the ferns should be big enough to cover some more dirt. but let me know what you think

**Comment by kaptain-kavern** [comment]
Would you want me to also keep the old plant graphics? That way the game will randomly use both graphics ? (like it's done for [Ferns](https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/tree/master/Textures/Things/Plant/Fern))

Ok doke I will make all of them bigger, it's in the code using _<visualSizeRange>_ tag

**Comment by lpCoercion** [comment]
no. we should replace them. i can edit these later if ee want more variety

**Comment by kaptain-kavern** [comment]
All right

**Comment by lpCoercion** [comment]
I tried to replace the textures on Github, but it keeps reverting back to the old upload.

**Comment by lpCoercion** [comment]
after testing a bunch last night, i think that the plants sizes are good as with maybe minor adjustments. if we make the plants too big everything looks really buy and messy

**Comment by lpCoercion** [comment]
I just made smokeleaf, cotton, healroot and psychoid a lot less common. So, you may only see a few during a playthrough.

I may also add more plants to their graphics like I did for the above.

**Comment by lpCoercion** [comment]
Lemongrass
![lemongrass](https://cloud.githubusercontent.com/assets/22029252/18619288/7b05853c-7dc6-11e6-9917-e0b6aec41c28.png)

Lemongrass Edible
![lemongrassfood](https://cloud.githubusercontent.com/assets/22029252/18619290/7ed7a44c-7dc6-11e6-84ee-9ae88e818e4b.png)

**Comment by lpCoercion** [comment]
Bamboo
![bamboo](https://cloud.githubusercontent.com/assets/22029252/18619419/88948ac4-7dc9-11e6-92d5-c475862e934a.png)

**Comment by lpCoercion** [comment]
Coconut Palm
![coconutpalm](https://cloud.githubusercontent.com/assets/22029252/18630207/ebecacc2-7e38-11e6-9458-1f7b0acff0b6.png)

**Comment by lpCoercion** [comment]
Coffea
![coffea](https://cloud.githubusercontent.com/assets/22029252/18630619/ce7c05ae-7e3b-11e6-9b1a-7ad98557d873.png)

**Comment by lpCoercion** [comment]
Heliconia
![heliconia](https://cloud.githubusercontent.com/assets/22029252/18632371/19085bb4-7e45-11e6-96d6-c9a918a834c2.png)

**Comment by lpCoercion** [comment]
Taro Root
![tarofood](https://cloud.githubusercontent.com/assets/22029252/18632852/8193065a-7e47-11e6-8f41-c3d864b90a5d.png)

Coffee Beans
![coffeebeans](https://cloud.githubusercontent.com/assets/22029252/18633110/adb8762e-7e48-11e6-948c-de5c958baeae.png)

Coconuts
![coconuts](https://cloud.githubusercontent.com/assets/22029252/18633465/66d76d26-7e4a-11e6-9f48-5be64cc39b9b.png)

Bananas
![bananas](https://cloud.githubusercontent.com/assets/22029252/18633659/5a9a8e66-7e4b-11e6-8ba0-fb7265169363.png)

**Comment by lpCoercion** [comment]
That should be all of the plants. Let me know when they all make it in game :)

**Comment by kaptain-kavern** [comment]
omg neat !

I just wake up here. Let me have my coffee (![coffeebeans](https://cloud.githubusercontent.com/assets/22029252/18633110/adb8762e-7e48-11e6-948c-de5c958baeae.png)) and it will be quick :stuck_out_tongue: 

Bamboo and Lemongrass are already in btw :wink:

**Comment by lpCoercion** [comment]
The rainforest looks great btw :)

I posted a screenshot to the forums.

to do list:
1. make the plants (ferns) slow down pawns considerably more. [done]
2. make trees take a lot longer to cut down, but provide a lot more wood. [done]
3. make rafflesia less common? [done]
4. make the predators hit harder, so that only a few hits can take out a pawn. [done]
5. remove the pathcost from the banana tree and make it an occupied square [done]
6. make the other plants take longer to cut down/clear. [done]
7. make the plants/trees provide a lot more cover. [done]
8. Is it considered dark when raining? If not, make it be considered dark for solar panels.

**Comment by kaptain-kavern** [comment]
Ok buddy, all new graphics are in :+1: 

I have some IRL matter to attend to and I came back to reply to your question. (all of it is doable except maybe rain/light relation)

Do you want me to explain where it is done in the code (for you to know it) or just that I do it?

**Comment by lpCoercion** [comment]
awesome :)

no worries. real life comes first.

I will take a look at the code in a bit. I may be able to make the changes myself. If not, I will let you know. I will work on testing today. If everything looks good, I will post v1.0 tonight :)

**Comment by lpCoercion** [comment]
can you also remove the pathcost from the banana tree and make it an occupied square?

**Comment by lpCoercion** [comment]
I did the ones that I was able to do. I am not sure what "combatpower" is in the predator defs though, so left it alone.

**Comment by lpCoercion** [comment]
I also dont see any heliconia in game

**Comment by kaptain-kavern** [comment]
> ## to do list:
> 
> **Make the plants (ferns) slow down pawns considerably more.**         

The value is defined with the _\<pathCost>_ tag : can look here https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/c71c6994d20f91fbc628221566bec211dccb3f7c

> **Make trees take a lot longer to cut down, but provide a lot more wood.**

The value is defined with the _\<harvestWork>_ tag : can look here https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/e4ab6049116408e242ce583023bf2c846fa249ca

> **Make the other plants take longer to cut down/clear.**

Same as above

> ~~**Make rafflesia less common?**~~ [done]

You already figured out this one.

> **Make the plants/trees provide a lot more cover.**

The value is defined with the _\<fillPercent>_ tag : It's defined at **0.43** (exactly like in Vanilla) in our _/Defs/ThingDefs_Plants/Plants_Bases.xml_ file (If you change it in this file, **then all of our trees** will inherit this value; else you can add the tag in each trees code - _in the Plants_Tropical.xml file_ - for individual different values.

> **Make the predators hit harder, so that only a few hits can take out a pawn.**

For now all I've done is copy/paste Vanilla values, so our felines are hitting exactly as hard as cougars/panthers in vanilla. (just for you to know, regarding on how you want to balance them with vanilla animals).  
The value are defined in each animal code. It is starting from <verbs> tags in the code. Each "paragraph"(starting by a `<li>` and closed by a `<\li>`) define one attack (like for felines : two "paragraphs", one for each front paws, and one other for a bite attack). And there you can change values. Note that it works like I explained above for plants and plantbases; if you change values for the bases (I always put them first on top of the files) all animals using that base will inherit those values; else you can define them for each animals (lower in the file) in their respective code part

> **Is it considered dark when raining? If not, make it be considered dark for solar panels.**

For this one i'll have to investigate a bit, but i'm pretty sure i've seen reference to that in the code. I'm not sure where :laughing: ... Maybe in biomes, or weather files ... I will search

**Comment by kaptain-kavern** [comment]
> **can you also remove the pathcost from the banana tree and make it an occupied square?**

In the code it set to `<passability>PassThroughOnly</passability>` so I was thinking it would count as occupied square? (But haven't particularly tested this)  
Does the Banana tree not behave like other trees?

**It does. I was confused because it has a pathcost**

**Comment by lpCoercion** [comment]
nvm.. it does. I was confused because it has a path cose

**Comment by lpCoercion** [comment]
I made all of the changes except for the solar panels in rain. I was told by someone on slack that rain already effects solar panels.

Also, I dont see any heliconia in game.

**Comment by kaptain-kavern** [comment]
Damn, too bad I don't have enough time to hang out on the Slack as before ... :cry: 

It's weird, because I see some : 
![Imgur](http://i.imgur.com/aThQ3d0.png)

Ho ! But there are not selectable (like grass and our Ferns and Bromeliad) want me to change that ? We could even make them plantable in pot! (I imagine potted Rafflesias for putting in prisoner rooms ^^)

**Comment by lpCoercion** [comment]
no its fine if they are there. I may have just not been able to find them. I will look again :)

**Comment by lpCoercion** [comment]
If you have time today do some testing. If everything is working good, I will post up v1 tonight. :)

**Comment by kaptain-kavern** [comment]
I'm doing it right now.

There is little things to polish (like Coffee and LemonGrass are just blend simple ingredient, it may be nice to complexify things a bit, like having a coffee to drink - coded like a drug like chocolate in Vanilla) but for the rest it will really be by playing that we may noticed things or have bug reports/suggestions from users. 
That for saying, to me it's like we can do the 1.0 release :tongue:

**Comment by lpCoercion** [comment]
I think we need to push out a quick patch (v1.0.2) to fix banana yield. Can you change the banana yield from 30 to around 8 or 12? Does this change seem resonable?

**Comment by kaptain-kavern** [comment]
I'm OK. Let's start with 12 and see how it goes :wink: 

---

**Edit :** Done :+1:

**Comment by kaptain-kavern** [comment]
Could you please look at #16 ? I have a little change in the code in mind but I want your opinion before doing anything

**Comment by lpCoercion** [comment]
awesome thanks.

i responded. i agree that the taro are way too overpopulated and ferns shoukd be more prevelant. I dont know the fix though.

**Comment by lpCoercion** [comment]
Plants to add:
    cocoa tree (plant)
    cocoa beans (ingredient)
    chile pepper (plant) [3 variations] 
 red chile
 green chile 
 yellow chile 
 vanilla (plant)
vanilla beans (ingredient)
tobacco (plant)
tobacco (ingredient)

**Comment by kaptain-kavern** [comment]
Ok doke. So I guess i'm waiting for more textures :p 

---

On an other note : I noticed our flowers have very low beauty value compared to vanilla potted ones : Bromeliad: 3 and Heliconia: 5 for Daylily: 24 and Rose: 14 ; but are in par with the wild ones Astragalus and Dandelions.

This brings me to the point : I find it could be nice to been able to put Tropical flowers in pots to embellish bases, what you think?

**Comment by lpCoercion** [comment]
i like the idea. can you get a screenshot? I am not sure if they will look good potted.

**Comment by kaptain-kavern** [comment]
2 ways I see for this : 
- Either we stick to Vanilla and let our wild tropical plants with their actual beauty value **AND** i make a copy for the potted version with beauty up to 24 like vanilla potted plants.
- Or we bump the beauty up to 24 like in Vanilla even for the wild ones and I just add the tag make them appear with pots

**Comment by lpCoercion** [comment]
maybe we should try 25 for heliconia and 15 for bromeliad even for the wild ones. and just add the tag to make them appear in pots. thoughts?

**Comment by kaptain-kavern** [comment]
Ok for me. 
It's the simplest solution regarding code ^^

**Comment by lpCoercion** [comment]
If the flowers look weird potted though, I may have to create a new texture for potted versions anyway though lol

**Comment by kaptain-kavern** [comment]
First Try
![Imgur](http://i.imgur.com/Raz2urh.png)

**Comment by kaptain-kavern** [comment]
Better now
![Imgur](http://i.imgur.com/dJzFyqw.png)

It's in dev ==> Commit : https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/4eea60fd4526cb4ea08871c6540de0cf4e572c6e

**Comment by lpCoercion** [comment]
looks pretty good. I will fix the texture tonight (to center them in the pot). It may be easier to just copy and paste the defs so we can make changes to the wild and potted separately.

maybe we should add more :P potted ferns?

**Comment by kaptain-kavern** [comment]
That's how I've done in the end, separate defs. 
Because they should behave differently than the wilds one. 

The plant that have 25 of beauty will have to be replanted twice more often than the one with 15

He no problem for adding more :-p

**Comment by lpCoercion** [comment]
could make the ferns 10 beauty and need to be replanted even less?

Also, thinking about this more, I think we should leave the wild flowers at their vanilla equivalent beauty

**Comment by kaptain-kavern** [comment]
Done.

Added potted ferns and changes wild bromeliads and Heliconias beauty to 6 and 7(still a bit more than Vanilla, because hey it's beautiful tropical flowers ;) )

**Comment by lpCoercion** [comment]
awesome. any potted fern screenshots?

SpaceDorf is worried about plant beauty. have you expierenced this too? If so, would raising ferns to beauty 1 solve the problem?

**Comment by kaptain-kavern** [comment]
[Beauty set to 1 for Palm, Fern, Taro and Lemongrass ](https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/81ba9dcf975bbcef4b429f96cba6c2950f99c4dc) on Dev.

![Imgur](http://i.imgur.com/n4V5PDd.jpg)

**Comment by lpCoercion** [comment]
very cool. do you think that the fern should be slightly bigger? does the pot use all three variations of ferns?

**Comment by kaptain-kavern** [comment]
In fact, we can play a bit with size in the code with `<visualSizeRange>` and for all potted versions of plant I had to set them to the same value (something I noticed in Vanilla code) for it looking better. I can try to increase ferns a bit.

Also as you can see on the first screenshot (above) , like vanilla flowers, I set them to been able to be planted in pot **AND** soil; except for ferns, I can't explain exactly why but that seemed weird to cultivate ferns in the forest ^^ - it can be changed too though

And yes the pot use all textures version. If I understand how things work correctly, the `<graphicClass>Graphic_Random</graphicClass>` tag in the code means : Use randomly anything find in the folder path (declared with `<texPath>` just above in the code)
![Imgur](http://i.imgur.com/eGtUrVv.png)

**Comment by lpCoercion** [comment]
These textures should fit the pots better. These are separate textures and do no replace the wild ones:

Potted Bromeliad
![pottedbromeliad](https://cloud.githubusercontent.com/assets/22029252/18810386/a26e4168-8262-11e6-994d-237ec781f0c0.png)

Potted Heliconia
![pottedheliconia](https://cloud.githubusercontent.com/assets/22029252/18810390/a5f2258e-8262-11e6-9100-6eb5740ccd15.png)

**Comment by kaptain-kavern** [comment]
Ok the potted ones are done. :+1: 

Also I've "tried something" just want to know your opinion.

I tried to make the Flowers planted in Crop to use the wild textures, (meaning that as they grow there will be several plants to flourish one the same spot). Just tell me what you think (I can't decide myself)

_(You can also see the potted with their new textures)_
![Imgur](http://i.imgur.com/af7ZaQo.png?1)

Or using the same textures as potted ones : 
![Imgur](http://i.imgur.com/n4V5PDd.jpg)

**Comment by lpCoercion** [comment]
That top screenshot is awesome. Good idea :)

Can you make the potted flowers a little bigger?

**Comment by kaptain-kavern** [comment]
I just upped them a bit, from

```
      <visualSizeRange>
        <min>0.9</min>
        <max>1.1</max>
      </visualSizeRange>
```

to

```
      <visualSizeRange>
        <min>1.2</min>
        <max>1.5</max>
      </visualSizeRange>
```

**Comment by lpCoercion** [comment]
alright cool. whatever looks good :)

**Comment by lpCoercion** [comment]
Can you remove the injestible tag from ferns? That way animals will eat taro and other edibles/harvestables instead.

**Comment by lpCoercion** [comment]
Or would this remove harvestables completely?

... maybe we make ferns non-injestable, but have shorter lifespans?

**Comment by kaptain-kavern** [comment]
For know I've done like in Vanilla, meaning all plants that is useful have a lower <ingestible> value than others (moss, grass, etc).

It was more because we haven't discuss it rather than because I wanted it that way.

I fear that completely removing the <ingestible> will break the ecosystem.

What we can try also is change those values.
Making fern values lower than other plants. That way I think animals will prioritize fruits-bearing and other "good plant" rather than ferns, but will not die of starvation in a the middle of ferns forest ;)

That way we don't introduce massive changes and I think reach the same goal you were aiming for

**Comment by lpCoercion** [comment]
alright cool.

**Comment by lpCoercion** [comment]
would this need to be pushed out in a patch?

**Comment by kaptain-kavern** [comment]
I haven't done it yet. But it can be include in next release. I also have to make some changes in plants (about blight incident resistance like discussed in the forum thread) so I will include both changes at once.

I also have all the code ready for Bongos, if needed for next release ;)

**Comment by lpCoercion** [comment]
oh ok cool. bongos should be ready for next release :)

**Comment by kaptain-kavern** [comment]
Ok I tried the changes discussed above and it's all working. Animals are not completely ignoring ferns but they make "efforts" to search and eat the good plants first. - _(This alone wasn't savecompatible)_

---

As changes above will require a new save. I also completely reworks some plants defs location in the different files (even created more files - the ones in **bold** below) : 
- **`Plants_Cultivated_Decorative.xml` contain the potted version of plants we add**
- **`Plants_Cultivated_Farm.xml` contain the crop version of plants we add**
- `Plants_Tropical.xml` now only contain the wild version of the modded plants we add
- `Plants_Vanilla.xml` contain the wild version of vanilla cultivated plants
- `Plants_Bases.xml` contain only the basedef all other plants are using

Two reasons : 
1. It was required to have cultivated plant not been able to spawn wild or the blight incident will not target them too (it is coded like that in C# - as Zhentar pointed it, [there](https://ludeon.com/forums/index.php?topic=26265.msg267757#msg267757))
2. It will help when we will introduce modular packs/mini mods. Also if someone have something he don't like it is easier to erase part of the mod.
   
   ---
   
   The only thing is that all of this is not save-compatible (So it's on a separate branch for now - https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/b0d35fd498257b26791af13f5035d100a1810409). So maybe there is no need to rush the release and wait for adding more. Either way it's OK with me, just something to consider
   
   ---
   
   I have the other ongoing changes (namely Bongo's and the optional changes to elephant for eating trees) on the Dev branch

**Comment by kaptain-kavern** [comment]
With the next released in mind (new plants, crops, recipes, etc),  I went nuts and [progressed on several "fronts" yesterday](https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commits/dev), here is an update because I want to know if it suits you : 
- https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/baf7f96934b8db5105a281d1aa6ed194b9e7f066 - Cigarettes added (need texture for plants at least, I managed a rolled cigarette textures by recoloring joints)
- https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/0e4271d3aae0701be157e14dc581e2446750d4a3 - All soil plants now have higher value (between 0.14 and 0.25) than ferns
  (set to 0.12) : Animal should prioritize eating "good" plants before
  ferns
- https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/b0d35fd498257b26791af13f5035d100a1810409 - Separate wild plants from potted and croped ones for the blight incident issue

**Comment by lpCoercion** [comment]
Can you list all of the changes so far for the next release 1.3?

**Comment by lpCoercion** [comment]
looks good :)

Can you please check out the checklist issue?

https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/issues/2#issuecomment-250930615

**Comment by lpCoercion** [comment]
Tree Stump
![treestump](https://cloud.githubusercontent.com/assets/22029252/19037159/cf1adeba-8942-11e6-83e0-9bb630cfb077.png)

Beehive
![beehive](https://cloud.githubusercontent.com/assets/22029252/19037162/d1837144-8942-11e6-9649-27f434c73de7.png)

Honeycomb
![honeycomb](https://cloud.githubusercontent.com/assets/22029252/19037164/d426ead4-8942-11e6-8b19-dbb35b2c6397.png)

Also, can you replace potted Heliconia with this texuture (the old one is not in the pot).

![pottedheliconia](https://cloud.githubusercontent.com/assets/22029252/19037269/57c73bc8-8943-11e6-8854-b9e9d6ce2751.png)

**Comment by lpCoercion** [comment]
I had another idea for Kava. It can be harvested for Kava Root. Kava Root can be used raw as a mild drug or refined at a drug lab to make Neutronamine.

**Comment by lpCoercion** [comment]
I hope your feeling better :)

I was wondering if you could add the tree stump in the same way that the beehive will be added. It would have the same stats as a strangler fig but produce a lot less wood and be very rare. this would add some more diversity. it could show up in game as "tree (dead)"
.

**Comment by lpCoercion** [comment]
apiary

![apiary](https://cloud.githubusercontent.com/assets/22029252/19407019/22596a32-9263-11e6-9efd-f1e03c4fa6b7.png)

We will have to see how it looks in game. I can make edits if needed. The apiary would only be available by farming. It would be "sowed" and "harvested" the same way any other plant would be. We can lock it behind research (beekeeping)

**Comment by lpCoercion** [comment]
I had an idea for "Kava" and "Kava Root"

Kava Root would produce effects similar to beer and can be synthesized at a drug lab to make Neutroamine

**Comment by kaptain-kavern** [comment]
So now about here ^^ : 
- Beehives are in place and are working (we may want to heavily test about yielding value and time to grow and such)
- I replaced potted Heliconia texture
- I haven't added the tree stumps as "dead tree" but i like it and it will not be a problem
- I like the tiny apiary ! I will work on adding it this night. I wonder how it will look inside a crop between "real plants"
- Okey Dokey for the Kava and root. Do we keep the Neutroamine crafting apart in the "F&D Pack" ?

**Comment by lpCoercion** [comment]
Awesome.

I could see it either way. It is a new recipe being added, but it is minor. Maybe we should just add it with the F&D Pack to be consistent?

**Comment by lpCoercion** [comment]
As discussed, "kava plant" and "kava root" will not be replaced with "dream vine" and "dream root"

Since we are putting the CK - F&D Pack on hold, a bill to turn dream root into neutroamine will need to be added to the drug lab in the main mod.

These are the updated textures:

dream vine:
![dream vine](https://cloud.githubusercontent.com/assets/22029252/19615939/aeff4fe8-97d5-11e6-89bd-64c5c87db037.png)

dream root:
![dream root](https://cloud.githubusercontent.com/assets/22029252/19615910/4743546c-97d5-11e6-91f6-137ea0693fd2.png)

I also want to retexture the taro plants with the following art:

![taro](https://cloud.githubusercontent.com/assets/22029252/19616136/9c17e3fa-97d9-11e6-89c1-bd3d6756ce23.png)

**Comment by kaptain-kavern** [comment]
About taro texture : Should I completely remove the other? ![tarob](https://cloud.githubusercontent.com/assets/2794612/19627221/44aee7b6-9942-11e6-8564-678d5ffcac63.png) or do we keep it for them to be randomly displayed? 

And also I noticed the picture's size is bigger. As is it will look bigger in game, have you got particular ideas/instructions on the displayed size?

_edit:_  
In fact without touching the code I find they look well : 
![clipboard](http://i.imgur.com/WVbMCqL.jpg?1)

And here it is with both Taro texture : for you to see how they mix : 
![Imgur](http://i.imgur.com/1RCCJ3E.png?1)

---

I know I keep saying it : but again I love the new arts! Very much like the "hand draw" style (I don't know the right words) I find it "fresh" but looking real, less "polished"/plastic/clean to my eyes than vanilla style

**Comment by lpCoercion** [comment]
thanks :)

my thought was to make them 128x128 and replace the other texture in game. that way it will cover more dirt and look less mechanically placed.

we could keep them the default size (as shown above) for crops though?

maybe you could take a screenshot of the bigger size so we can cimpare them side by side?

either way, I think we should completely replace the original texture though.

What are your thoughts?

**Comment by kaptain-kavern** [comment]
Oh right I understand why bigger then.       

As it is done ATM, wild and cultivated one are displayed the same. I will make the wild ones bigger and make another screenshot. _(here is a bigger one, the wild, up left - I kept the battery to help appreciate the size better)_
![Imgur](http://i.imgur.com/KqQyTll.png?1)
I think we should only keep the new one, also.     

About looking less "artificial", I know it is more works for you, but it is possible to use several textures for the same plant and the game will randomly display them. But it's not a priority I think :-p

---

Here are the Apiaries : 
![Imgur](http://i.imgur.com/vfs1vOj.png?1)

**Comment by lpCoercion** [comment]
do you think that the apiaries are to big? from that screenshot, they look to be almost as tall as a tree and bigger than a grave. maybe they should be 1/4 that size?

**Comment by lpCoercion** [comment]
awesome :)

the apiaries look good in that screenshot. how do they look in game?

**Comment by kaptain-kavern** [comment]
I posted this one on the forum : ![](http://i.imgur.com/xNThg1n.png?1)

I thought about their size. I was thinking they will look way too big and all that, but after seeing them in game (and despite the fact I completely know and see they're bigger than realist size) i find them quite good like that. But hey it's you the artist ;) you have the eye :p

**Comment by lpCoercion** [comment]
oh ok. well we can always change it in the future

**Comment by kaptain-kavern** [comment]
## Need your thought on the Dream Vine

_About how their size? How tall they're supposed to be ? more like a bush?_ 
Wild Dream Vine (too small don't you think?)
![Imgur](http://i.imgur.com/ooOF4QU.png?1)![Imgur](http://i.imgur.com/0toyVkM.png?1)

Cultivated one (a bit bigger but still to small)
![Imgur](http://i.imgur.com/386hFj8.png?1)![Imgur](http://i.imgur.com/Z9JGtU0.png?1)

**Comment by lpCoercion** [comment]
2i think the second picture looks good (about the size of an okapi).

I can create another texture for the cultivated version. dont let that hold up the release rhough. we can just use the one texture for both the wild and crop version for noq amd. then we can push out a quick release to  add another texture for crops

**Comment by kaptain-kavern** [comment]
All right then. 
In the biomedef I set the wild dream vines to `"0.2"` regarding commonalities for now. 

sounds good.

The raw dream root should induce 'Alcohol effect/drunk' like beers, but less strong, right ?        

yeah.

Also can the root be used as cooking ingredient as well? (I was wondering about that for the raw coffee too - _It could force the player to have to choose between : reserve it for the "chemical"/induced" effect OR use it as cooking ingredient_ - by forbid or not in recipes/stockpiles settings).

I like this idea. They can be used as raw ingredients for meals too.

I'm not exactly sure on the detail for the Neutroamine crafting. It should be expensive right? like 10/15 roots for 1 neutroamine? _We can also plays with the harvested amount by plants, duration of crafting process and such for balancing the difficulty_

sounds good to me. maybe start with 10 per neutro.. duration could be a bit longer than medicine crafting?

**Comment by lpCoercion** [comment]
I commented in your post above. I like all the ideas.

**Comment by kaptain-kavern** [comment]
Then it's nearly the end ^^ :wine_glass: 
All is done, I'm just finishing the Neutroamine crafting recipe ATM (do we want a mini skill required for it?). And the all thing will needs a bit of testing I'd say :wink: 

If you put this recipe in? do we add another one for the raw coffee to be of a purpose BTW?

And, last question, do you have a Bongo texture I've missed? For ~~k~~now they are still displayed as big gazelle

**Comment by lpCoercion** [comment]
neutromine should maybe need a skill of 8 or 10? thoughts?

i think we should wait on adding a coffee recipe and just have coffee beans either be eaten raw as a drug or used in cooking as a raw plant.

no bongo yet. just remove it from game until i get the texture done.

**Comment by kaptain-kavern** [comment]
As it is now, the recipe require a level 10 in cooking, tell me if I change.

Okey dokey for the coffee cups recipe (Coffee beans are already working like that, right now) and I remove/comment the Bongos code

**Comment by lpCoercion** [comment]
should the recipe require 10 craft? isnt craft what all the drugs use?

**Comment by kaptain-kavern** [comment]
I just checked in game an for crafting all drugs there is no skill requirement at all and the working skill is `<workSkill>Crafting</workSkill>`. It's just for crafting medecine that there is a requirement of 3 in crafting and 6 in medecine.

As a trade of i've set the requirement to be 

```
<skillRequirements>
      <li>
        <skill>Cooking</skill>
        <minLevel>3</minLevel>
      </li>
      <li>
        <skill>Medicine</skill>
        <minLevel>6</minLevel>
      </li>
</skillRequirements>
```

**Comment by kaptain-kavern** [comment]
I also have made little improvements while at it ^^

## QoL improvement
- Coffee Beans and Dream Vine Root can be schedule like drugs (Can be
  used anyhow but at first I was thinking like : Take 1 shot of coffee
  everyday at 17H, per example; OR the possibility to just add Dream Vine
  Roots in "Social Drugs" as Alcohol)
  ![Imgur](http://i.imgur.com/3JBMhtx.png?1)
- RawCoffee effect is now more noticeable by the player (added a label
  in Health Tab + a "Thought" in mood panel)
  ![Imgur](http://i.imgur.com/XCkx3Go.png?2)
- Add in game display : "XXXX take Coffee Beans"/"XXXX is taking Dream
  Vine Roots
  ![Imgur](http://i.imgur.com/Q1Lhghp.png?1)

More screenshots : 
![Imgur](http://i.imgur.com/hR8UD6Q.png?2) /

**Comment by lpCoercion** [comment]
awesome:)

some thoights:
- can we make the requirement craft instead of cooking? that way it allows crafters who make drugs to do it. maybe the craft skill should be higher though? maybe craft 6? aso should the medicine requirement be removed since neutro is used for other drugs too?
- the mood descriptiom should say: "Those coffee beans... tastes like s%$#, but man are they strong." If you dont like that I can rewrite it but the original grammar is wrong.
- maybe instead of "taking" coffee beans it says "ingesting"? taking sounds like hauling to me

**Comment by lpCoercion** [comment]
also. maybe you are right and we should add the coffee recipe to the drug lab to remove the taste debuff of coffee beans?

**Comment by lpCoercion** [comment]
I noticed that the research for beekeeping is 500. is there an advantage to apiarys over other crops that makes it require a long research?

**Comment by kaptain-kavern** [comment]
Oh you know I kinda put that number like that. I'm afraid all the plants need a real long thinking and balancing in fact ^^  
I mean, we "just" add them as ideas came but I never take some times to consider all as a whole and compare them between each other, etc...

In order to be able to say which should yield better, with which soil quality importance, those kind of things.

Per example, I think trees (and in a way apiaries also) should take longer to grow but in exchange should yield better than short time plants. 

But for all this it's always the same we need Vanilla data I think.  
I will put a table together with that (I particularly want yielding/day, etc); so we can balance ours on par with vanilla; and we will talk about it again :wink: (I have started the wiki a bit - will put the data there)

**Comment by lpCoercion** [comment]
sounds good

do you have link to the wiki?

**Comment by lpCoercion** [comment]
also... i do like the apiary size. l will create a new texture though to replace it.

**Comment by kaptain-kavern** [comment]
Here it is 

![Imgur](http://i.imgur.com/eM4aRo8.png?3)

**Comment by lpCoercion** [comment]
thanks :)

**Comment by kaptain-kavern** [comment]
https://docs.google.com/spreadsheets/d/1IueAG4N5ulB17FPA0xi046MSpyOFSH53InqRNF2yrEg/edit?usp=sharing

I send you an invit' for the spreadsheet

---

I haven't changed any value. just by looking at the yellow column you can see what I was afraid of, it is clearly not balanced ^^  Nothing is matching Vanilla value pattern.

Please experiment with the numbers to see how you feel it. (Yellow columns are automatically computed). I will do the same.

The way I see we have to "play" with _harvestYield_, _growDays_ and the price of the fruits/harvested item (_price per unit_)

_fertility Factor Growth Rate_ could serve as a added bonus/malus for balancing a bit more. I understand it as : "the way soil-quality impact the growing time of the plant"

**Comment by lpCoercion** [comment]
cool :) I will work on it now

should do another one for animals too

**Comment by kaptain-kavern** [comment]
I completely finish the spreadsheet, for cultivated and wild plants.
I will make a copy (for backup) and will play with numbers a bit, and come back at you with my thoughts

**Comment by lpCoercion** [comment]
oh ok cool :)

**Comment by kaptain-kavern** [comment]
So here are my first thoughts after some thinking on the cultivated plants (only) : 

| label | old yielding | proposed new yielding | old \<growDays> | proposed new \<growDays> | old \<sowMinSkill> | proposed new \<sowMinSkill> |
| :-: | :-- | :-- | :-: | :-: | --: | --: |
| taro | 10 | 10 | 4 | 3,5 | None | None |
| lemongrass | 10 | 7 | 3 | 2,5 | None | None |
| dream vine | 6 | 10 | 6,84 | 6,84 | 8 | 6 |
| coffea bush | 30 | 10 | 5,5 | 5 | None | 4 |
| apiary | 40 | 60 | 25 | 25 | None | 7 |
| coconut palm tree | 10 | 45 | 18 | 18 | None | 5 |
| banana tree | 12 | 50 | 17 | 17 | None | 5 |
- **taro** : Slightly reduced `<growDays>` to be just better than Potato and reduced `<fertilityFactorGrowthRate>` to compensate
- **lemongrass** : Slightly reduced `<growDays>` to be just better than Rice and reduced `<fertilityFactorGrowthRate>` to compensate
- **dream vine** : Reduced min skill requirement and increased yielding; to be on par with Psychoid plant
- **coffea bush** : Adding Min skill requirement of 4 and reduced yielding; to be on par with Smokeleaf plants - Also need to reduce the price of coffee beans (with the previous stat it was netting 25 silver per tile a day)
- **apiary** : Increased yielding even more (not a problem because of the long growing time) and reduced price of honeycomb - I also find/feel it need to have a min skill Requirement, choose 7 like for hop plant
- **coconut palm tree** : Increased yielding vastly to compensate the long growing time.
- **banana tree** : Increased yielding vastly to compensate the long growing time.

Here is the link to the spreadsheet with the corrected values : https://docs.google.com/spreadsheets/d/1Bsh-yRx2iG3FNABDx0ekxuFC_i_kol0sowsNY9d43Ok/edit?usp=sharing

I have only done the cultivated plants one, for now

---

What do you think ?

---

**Comment by lpCoercion** [comment]
a couple things:
- we reduced the banana yield to 12 before because wild food was too common. not sure if that is still the case?
- will pawns need the min sow skill to harvest wild plants? if so, we may want to remove it from the tropical food plants (banana & coconut)

also, should honeycomb give a small mood buff to compensate for the time and research req?

**Comment by kaptain-kavern** [comment]
All the changes I proposed were just for cultivated plants only. I don't find too strange that cultivated species could yield more than wild ones, we could say the cultivated one is the result of selection.      

I'm pretty sure the min skill requirement are for crop plants only but as we have made them use two separate sets of code, I can have different settings for both version.

_will continue this post later - no battery left on phone :-)_

**Comment by lpCoercion** [comment]
oh ok i like it for crops only then :)

could include this in v1.3.1

**Comment by kaptain-kavern** [comment]
I just checked for honeycomb and yes they behave exactly as insect Jelly.

Okey dokey so I will make so change for cultivated ones then. And if I managed to not fall asleep I will look out for balancing the wild ones ;-)

**Comment by lpCoercion** [comment]
oh ok cool :)

I was also given the suggestion to make jungle plants less flammable than others because they are so wet. I think it makes sense.

**Comment by lpCoercion** [comment]
Coffea (crop)

![coffeacrop](https://cloud.githubusercontent.com/assets/22029252/20056112/8d974146-a4b2-11e6-810f-b074e660bb2a.png)

Dream Vine (crop)

![dream vine crop](https://cloud.githubusercontent.com/assets/22029252/20056552/d8a81492-a4b4-11e6-9b6b-29f755005e35.png)

**Comment by kaptain-kavern** [comment]
They're nice! I very like how the flower color seems to be more visible on the cropped Dream Vine. 

Also added. 
![Imgur](http://i.imgur.com/G2Cm3JF.png?1)
_I may try to add them for being randomly displayed, alternatively with the ones already in, for the wild ones. For testing only, doing a screenshot and show you_

**Comment by lpCoercion** [comment]
looks good :)

**Comment by lpCoercion** [comment]
jitter berries

![jitterberries](https://cloud.githubusercontent.com/assets/22029252/20224161/1c925308-a80b-11e6-94fa-942629a595a9.png)

**Comment by kaptain-kavern** [comment]
![Imgur](http://i.imgur.com/8m9BsAb.png?1) // ![Imgur](http://i.imgur.com/TZieWYo.png?1) // ![Imgur](http://i.imgur.com/E2UwhNF.png?1)

**Comment by lpCoercion** [comment]
looks good :)
